private/bufpkg/bufpluginexec/bufpluginexec.go

```diff
@@ -140,6 +144,12 @@ func NewHandler(
 		}
 		return newBinaryHandler(runner, pluginPath, handlerOptions.pluginPath[1:]), nil
 	}
+	if isWASM(pluginName) {
```

Comment:
Are we going to support executing remote wasm plugins or just local paths? If we want to start with local only, then we will want to do more path validation here.